core/src/components/popover/popover-interface.ts

```diff
@@ -29,7 +29,7 @@ export interface PopoverOptions<T extends ComponentRef = ComponentRef> {
   dismissOnSelect?: boolean;
   reference?: PositionReference;
   side?: PositionSide;
-  align?: PositionAlign;
+  alignment?: PositionAlign;
```

Comment:
You will also need to update the interface in `README.md` so the changes get reflected on the docs: https://ionicframework.com/docs/api/popover#interfaces

Reply:
Good catch 👍  wonder how common this is (referencing public interfaces manually). May be something we consider using MDX with a component to raw-load a source file or something 🤔